config.js

```diff
@@ -149,10 +149,17 @@ module.exports = {
       "url": domain + "/admin/suppliers?supplier_name_prefix=a",
       "user": "admin",
     },
+
     {
-      "label": environment + ": Admin - Account - Download user lists",
-      "url": domain + "/admin/users/download",
-      "user": "admin",
+      "label": environment + ": Category - Account - dashboard",
+      "url": domain + "/admin",
+      "user": "category",
```

Comment:
It would probably be good to call these `admin-category` and `admin-sourcing` to be more explicit.

Reply:
Will do.  I reckon also change the `admin` to `admin-support` as that's what we're calling them now in the frontend.